fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -561,6 +561,15 @@ alter_stmt ::=
     :}
     ;
 
+// Alter Load Statement
+alter_stmt ::=
```

Comment:
Adding a alter statement will add a new object in the RBAC model. So the statement should be discussed PM @wangsimo0